gallery/src/data-visualization/ReactVisDonutChartPage.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { GalleryDescriptionTile, GalleryExampleTile } from '../gallery-components/GalleryTiles';
+import ReactVisDonutChartExample from './examples/ReactVisDonutChartExample';
+import { NxTable, NxTableHead, NxTableRow, NxTableCell, NxTableBody } from '@sonatype/react-shared-components';
+
+const reactVisDonutChartExampleCode = require('./examples/ReactVisDonutChartExample?raw');
+
+const ReactVisDonutChartPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <p className="nx-p">
+        Donut charts are a type of <code className="nx-code">{'<RadialChart />'}</code> in React-vis. Both pie charts
+        and donut charts are supported by React-vis.
+      </p>
+      <p className="nx-p">
+        The following documentation only contains basics on how to get started with creating basic donut charts. <br />
+        For the full, official API documentation, please click {' '}
+        <a href="https://uber.github.io/react-vis/documentation/other-charts/radial-chart" target="_blank">here.</a>
+      </p>
+
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Data Format Reference</h3>
+        </header>
+
+        <p className="nx-p">
+          Data points for donut charts utilize the <code className="nx-code">RadialChartPoint</code>
+          {' '} interface.
+        </p>
+
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>angle</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                This determines the angular size of each wedge.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>color</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
```

Comment:
This doesn't appear to be required

Reply:
> This doesn't appear to be required

It's not required per the library, but I figured I would mark this as required so that users would be forced to use RSC colors. If `color` is not specified, react-vis creates charts with its default colors.

Reply:
> It's not required per the library, but I figured I would mark this as required so that users would be forced to use RSC colors. If color is not specified, react-vis creates charts with its default colors.

The problem with that is that there are a number of different ways to set the color, and this isn't necessarily the preferred one.  Which leads to something else I've been meaning to mention, I think we need to do significantly more to document and demonstrate how custom classnames would be added and used in order to color the various chart components - as that would be my preferred way of doing so.